lib/puppeteer-helpers.js

```diff
@@ -97,9 +97,39 @@ function goToComponent (page, componentName, { exampleName } = {}, waitForOption
   return goTo(page, componentPath, waitForOptions)
 }
 
+/**
+ * Wait for selector to be hidden or removed from the DOM
```

Comment:
Appreciate these functions existed already and you've just moved them, but I'm wondering how useful they are as an abstraction…

Should we consider just using `page.waitForSelector` in the tests? 🤔

Reply:
Yep I'm happy to do this

I can move the timeout option into `page.setDefaultTimeout(1000)` too?